EnglishTranslation-master/QUEST_LV_0100.tsv

```diff
@@ -12237,8 +12237,8 @@ QUEST_LV_0100_20151224_012237	Bishop Urbonas is the pillar sustaining the people
 QUEST_LV_0100_20151224_012238	I've been asking around to try to figure out what happened but it seems as if the priests have all dissappeared as well.{nl}
 QUEST_LV_0100_20151224_012239	My men are all looking for the whereabouts of Bishop Urbonas.. {nl}But it's pretty much a dead-end at this point.{nl}
 QUEST_LV_0100_20151224_012240	If you're as good as the rumors make you out to be, you'll have no trouble in finding where Bishop Urbonas is.{nl}Just make sure nobody knows about this.. The lord doesn't want the people being any more agitated than they already are.
-QUEST_LV_0100_20151224_012241	Could you ask that person how much longer he is planning on making us stand here?
-QUEST_LV_0100_20151224_012242	It's been days since we've arrived here, and yet we still haven't set foot in Orsha.{nl}Why did they make us travel all this way just to make us stand around like this?
+QUEST_LV_0100_20151224_012241	Could you ask that person how much longer is he planning on making us stand here?
```

Comment:
Those lines are from people that are just complaining because they are on a line, and that person is actually besides you.

![image](https://cloud.githubusercontent.com/assets/11585030/13547616/adaf18c6-e2b8-11e5-95c1-986ca3673736.png)

I don't think that it needs the quotes since it looks ok within the context